jetstream/tests/test_statistics.py

```diff
@@ -160,6 +160,58 @@ def test_binomial_pairwise_branch_comparisons(self, experiments):
             comparison_branches, key=lambda c: (str(c[0]), str(c[1]), str(c[2]))
         )
 
+    def test_bootstrap_mean_pairwise_branch_comparisons(self, experiments):
```

Comment:
If this is literally the same test with the first line changed, would it make sense to just put a loop around the existing test? Possibly make a fixture of multiple stats (could even test more/all of them)?

Reply:
good idea, let me add that